src/main/java/mServer/crawler/sender/br/WebAccessHelper.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * WebAccessHelper.java
+ * 
+ * Projekt    : MLib
+ * erstellt am: 04.10.2017
+ * Autor      : Sascha
+ * 
+ */
+// Ist eigentlich aus MLib in der neuen Architektur, aber für den neuen BR-Crawler erstmal hierher kopiert
+package mServer.crawler.sender.br;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)